src/mainwindow.cpp

```diff
@@ -587,6 +589,13 @@ void MainWindow::actionOpen()
     if (!fileName.isEmpty()) {
         m_lastFolder = QFileInfo(fileName).absoluteDir().absolutePath();
         openFile(fileName);
```

Comment:
Please leave a blank line and add a comment about what you are about to do (start a log file)